src/main/kotlin/com/example/ajaxproject/dto/request/ChatDTO.kt

```diff
@@ -1,6 +1,12 @@
 package com.example.ajaxproject.dto.request
 
-data class ChatDTO(
+
```

Comment:
Lol, empty line

Reply:
Please, don't use `Lol` or similar words. It could demotivate author

Better message:
> Please, remove redundant empty line